tuner/bayes_opt_gp/dffml_tuner_bayes_opt_gp/tests/test_regressor_model.py

```diff
@@ -0,0 +1,100 @@
+import pathlib
+import tempfile
+
+import numpy as np
+
+from dffml.record import Record
+from dffml.source.source import Sources
+from dffml import train, score, predict, run_consoletest
+from dffml.util.asynctestcase import AsyncTestCase
+from dffml.feature.feature import Feature, Features
+from dffml.accuracy import MeanSquaredErrorAccuracy
+from dffml.source.memory import MemorySource, MemorySourceConfig
+
+
+from dffml_model_xgboost.xgbregressor import (
+    XGBRegressorModel,
```

Comment:
Was this file left in intentionally? Let's double check for other additions like this which might be from directory copy pastin getc.

Reply:
Yes, you're right. I'll remove it.